api/src/main/java/org/eclipse/microprofile/metrics/annotation/package-info.java

```diff
@@ -84,7 +83,24 @@
  *      MetricRegistry appRegistry;
  * </code>
  * </pre>
+ * 
+ * <h2>RegistryType CDI Qualifier</h2>
+ * <p>
+ * 
+ * <b> This is DEPRECATED. Please use {@link RegistryScope} instead. </b>
+ * 
+ * The {@link org.eclipse.microprofile.metrics.annotation.RegistryType RegistryType} is used to identify which
+ * <code>MetricRegistry</code> (Application, Base, or Vendor) should be injected. By default, no
```

Comment:
wording above, for RegistryScope is a bit better.  "Injecting a MetricRegistry without a RegistryType annotation gives the application-scoped..."